<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.apache.kafka.raft.KafkaNetworkChannelTest" tests="24" skipped="0" failures="0" errors="0" timestamp="2024-04-09T11:20:19" hostname="ip-192-168-1-113.eu-central-1.compute.internal" time="3.094">
  <properties/>
  <testcase name="testWakeupClientOnSend()" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="1.307"/>
  <testcase name="testSendToBlackedOutDestination()" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.06"/>
  <testcase name="testUnsupportedVersionError()" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.078"/>
  <testcase name="testSendAndFailAuthentication()" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.021"/>
  <testcase name="testSendAndReceiveOutboundRequest()" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.003"/>
  <testcase name="testSendAndDisconnect()" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.001"/>
  <testcase name="[1] version=0" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.122"/>
  <testcase name="[2] version=1" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.011"/>
  <testcase name="[3] version=2" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.266"/>
  <testcase name="[4] version=3" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.018"/>
  <testcase name="[5] version=4" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.002"/>
  <testcase name="[6] version=5" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.009"/>
  <testcase name="[7] version=6" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.092"/>
  <testcase name="[8] version=7" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.037"/>
  <testcase name="[9] version=8" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.02"/>
  <testcase name="[10] version=9" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.028"/>
  <testcase name="[11] version=10" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.001"/>
  <testcase name="[12] version=11" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.052"/>
  <testcase name="[13] version=12" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.0"/>
  <testcase name="[14] version=13" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.005"/>
  <testcase name="[15] version=14" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.003"/>
  <testcase name="[16] version=15" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.008"/>
  <testcase name="[17] version=16" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.008"/>
  <testcase name="testSendToUnknownDestination()" classname="org.apache.kafka.raft.KafkaNetworkChannelTest" time="0.004"/>
  <system-out><![CDATA[[2024-04-09 14:20:20,590] ERROR Request OutboundRequest(correlationId=0, data=VoteRequestData(clusterId='clusterId', topics=[TopicData(topicName='topic', partitions=[PartitionData(partitionIndex=0, candidateEpoch=5, candidateId=1, lastOffsetEpoch=4, lastOffset=329)])]), createdTimeMs=1712661620557, destinationId=2) failed due to unsupported version error (org.apache.kafka.raft.KafkaNetworkChannel:128)
org.apache.kafka.common.errors.UnsupportedVersionException: Api VOTE with version 0
[2024-04-09 14:20:20,629] ERROR Request OutboundRequest(correlationId=1, data=BeginQuorumEpochRequestData(clusterId='clusterId', topics=[TopicData(topicName='topic', partitions=[PartitionData(partitionIndex=0, leaderId=1, leaderEpoch=5)])]), createdTimeMs=1712661620557, destinationId=2) failed due to unsupported version error (org.apache.kafka.raft.KafkaNetworkChannel:128)
org.apache.kafka.common.errors.UnsupportedVersionException: Api BEGIN_QUORUM_EPOCH with version 0
[2024-04-09 14:20:20,630] ERROR Request OutboundRequest(correlationId=2, data=EndQuorumEpochRequestData(clusterId='clusterId', topics=[TopicData(topicName='topic', partitions=[PartitionData(partitionIndex=0, leaderId=5, leaderEpoch=1, preferredSuccessors=[2])])]), createdTimeMs=1712661620557, destinationId=2) failed due to unsupported version error (org.apache.kafka.raft.KafkaNetworkChannel:128)
org.apache.kafka.common.errors.UnsupportedVersionException: Api END_QUORUM_EPOCH with version 0
[2024-04-09 14:20:20,634] ERROR Request OutboundRequest(correlationId=3, data=FetchRequestData(clusterId=null, replicaId=-1, replicaState=ReplicaState(replicaId=1, replicaEpoch=-1), maxWaitMs=0, minBytes=0, maxBytes=2147483647, isolationLevel=0, sessionId=0, sessionEpoch=-1, topics=[FetchTopic(topic='topic', topicId=tlm3H9PyTr-TLlWZBaprsA, partitions=[FetchPartition(partition=0, currentLeaderEpoch=5, fetchOffset=333, lastFetchedEpoch=5, logStartOffset=-1, partitionMaxBytes=0)])], forgottenTopicsData=[], rackId=''), createdTimeMs=1712661620557, destinationId=2) failed due to unsupported version error (org.apache.kafka.raft.KafkaNetworkChannel:128)
org.apache.kafka.common.errors.UnsupportedVersionException: Api FETCH with version 16
[2024-04-09 14:20:20,664] ERROR Request OutboundRequest(correlationId=0, data=VoteRequestData(clusterId='clusterId', topics=[TopicData(topicName='topic', partitions=[PartitionData(partitionIndex=0, candidateEpoch=5, candidateId=1, lastOffsetEpoch=4, lastOffset=329)])]), createdTimeMs=1712661620640, destinationId=2) failed due to authentication error (org.apache.kafka.raft.KafkaNetworkChannel:135)
org.apache.kafka.common.errors.AuthenticationException: Authentication failed
[2024-04-09 14:20:20,665] ERROR Request OutboundRequest(correlationId=1, data=BeginQuorumEpochRequestData(clusterId='clusterId', topics=[TopicData(topicName='topic', partitions=[PartitionData(partitionIndex=0, leaderId=1, leaderEpoch=5)])]), createdTimeMs=1712661620640, destinationId=2) failed due to authentication error (org.apache.kafka.raft.KafkaNetworkChannel:135)
org.apache.kafka.common.errors.AuthenticationException: Authentication failed
[2024-04-09 14:20:20,666] ERROR Request OutboundRequest(correlationId=2, data=EndQuorumEpochRequestData(clusterId='clusterId', topics=[TopicData(topicName='topic', partitions=[PartitionData(partitionIndex=0, leaderId=5, leaderEpoch=1, preferredSuccessors=[2])])]), createdTimeMs=1712661620640, destinationId=2) failed due to authentication error (org.apache.kafka.raft.KafkaNetworkChannel:135)
org.apache.kafka.common.errors.AuthenticationException: Authentication failed
[2024-04-09 14:20:20,666] ERROR Request OutboundRequest(correlationId=3, data=FetchRequestData(clusterId=null, replicaId=-1, replicaState=ReplicaState(replicaId=1, replicaEpoch=-1), maxWaitMs=0, minBytes=0, maxBytes=2147483647, isolationLevel=0, sessionId=0, sessionEpoch=-1, topics=[FetchTopic(topic='topic', topicId=V18BPcwBQm24ACtGIhhdJA, partitions=[FetchPartition(partition=0, currentLeaderEpoch=5, fetchOffset=333, lastFetchedEpoch=5, logStartOffset=-1, partitionMaxBytes=0)])], forgottenTopicsData=[], rackId=''), createdTimeMs=1712661620640, destinationId=2) failed due to authentication error (org.apache.kafka.raft.KafkaNetworkChannel:135)
org.apache.kafka.common.errors.AuthenticationException: Authentication failed
api key VOTE, response VoteResponseData(errorCode=42, topics=[TopicData(topicName='topic', partitions=[PartitionData(partitionIndex=0, errorCode=0, leaderId=5, leaderEpoch=1, voteGranted=false)])])
api key BEGIN_QUORUM_EPOCH, response BeginQuorumEpochResponseData(errorCode=42, topics=[])
api key END_QUORUM_EPOCH, response EndQuorumEpochResponseData(errorCode=42, topics=[])
api key FETCH, response FetchResponseData(throttleTimeMs=0, errorCode=42, sessionId=0, responses=[], nodeEndpoints=[])
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
